ui/full-page/import-recovery-phrase/ImportRecoveryPhrasePage.tsx

```
@@ -0,0 +1,234 @@
+    });
+  };
+
+  const handlePasteAll = async () => {
+    try {
+      const text = await navigator.clipboard.readText();
+      const parsed = text.trim().split(/\s+/);
+      if (parsed.length === 12 || parsed.length === 24) {
+        setPhraseLength(parsed.length as PhraseLength);
+        setWords([...parsed, ...Array(24 - parsed.length).fill("")]);
+      }
+    } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)